docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,87 @@
+
+For an introduction to native histograms, watch the following presentation: [Native Histograms in Prometheus](https://www.youtube.com/watch?v=AcmABV6NCYk).
+
+## Pros and cons
+
+This section describes the pros and cons of using native histograms compared to the classic Prometheus histograms. For more information and real example, see [Prometheus Native Histograms in Production](https://www.youtube.com/watch?v=TgINvIK9SYc&t=127s) video.
+
+### Pros
+
+1. Simpler instrumentation. No need to think about bucket boundaries, they are automatic.
+1. Better resolution in practice as custom bucket layouts are usually not that high resolution.
+1. Native histograms are compatible with each other due to the automatic layout, thus they can be easily combined. Caveat: the operation might scale down an operand to lower resolution to match the other operand.
+
+### Cons
+
+1. Loses precision in case the observed values are on a logarithmic scale already (e.g. sound pressure measured in decibels).
+1. Loses precision if you are converting from an external representation that has stict custom bucket boundaries.
+1. No way to set the bucket boundaries, which means that if a query falls between bucket boundaries, the result will be an approximation. In general this should not be a problem due to the much better resolution.
+
+## Use native histograms
+
+### Instrument application
+
+TODO: should note that one can keep the old histograms
+TODO: add GoLang examples. And maybe Python? Or link to examples.
+
+### Scrape native histograms with Prometheus
+
+TODO: should note that one can scrape both optionally
+
+### Scrape native histograms with Grafana Agent
+
+TODO: should note that one can scrape both optionally
+
+### Send native histograms to Grafana Cloud with Prometheus
+
+### Send native histograms to Grafana Cloud with Grafana Agent
+
+## Migrate from classic histograms
+
+### Instrument application
+
+### Scrape metrics with Prometheus
+
+TODO: keep scraping both
+
+### Scape metrics with Grafana Agent
+
+TODO: keep scraping both
+
+## Bucket boundary calculation
+
+Native histogram samples have three different kind of buckets, for any observed value `v` the value is counted towards one bucket.
+
+1. Zero bucket, which contains the count of observations whose absolute value is smaller or equal to the zero threshold.
+
+  [//]: # "LaTeX equation source: -ZT <= v <= ZT"
+   ![Zero threshold definition](zero-threshold-def.svg)
+
+1. Positive buckets, which contain the count of observations with a positive value that is greater or equal to the lower bound and smaller than the upper bound of a bucket.
+
+1. Negative buckets, which contain the coint of observations with a negative value that is smaller or equal to the upper bound and larger than the lower bound of a bucket.
```

Comment:
```suggestion
- A zero bucket, which contains the count of observations whose absolute value is smaller or equal to the zero threshold.

  [//]: # "LaTeX equation source: -ZT <= v <= ZT"
   ![Zero threshold definition](zero-threshold-def.svg)

- Positive buckets, which contain the count of observations with a positive value that is greater or equal to the lower bound and smaller than the upper bound of a bucket.

- Negative buckets, which contain the coint of observations with a negative value that is smaller or equal to the upper bound and larger than the lower bound of a bucket.
```
Ordered lists are for ordered items; use a bulleted list instead and trust that the reader can count. :)